client/scss/components/_footer.scss

```diff
@@ -166,3 +166,58 @@
     }
   }
 }
+
+// Footer control bar for performing actions on the page
+footer .actions,
+footer .preview {
+  .button {
+    @apply w-leading-none w-inline-flex w-items-center;
+
+    .icon {
+      margin-inline-end: theme('spacing.2');
+    }
+  }
+}
+
+footer .actions {
+  .button {
+    font-weight: 600;
```

Comment:
Should we use the theme font weight variable here?

Reply:
I’d rather not change this component at all if possible. The only difference is that it now gets its own file, but it does need much more refactoring.